dotnet/packages/Microsoft.Bot.Builder.M365/Microsoft.Bot.Builder.M365/AI/OpenAI/OpenAIClient.cs

```diff
@@ -32,17 +33,21 @@ public OpenAIClient(OpenAIClientOptions options, ILogger? logger = null, HttpCli
         /// Make a call to the OpenAI text moderation endpoint.
         /// </summary>
         /// <param name="text">The input text to moderate.</param>
+        /// <param name="model">The moderation model to use.</param>
         /// <returns>The moderation result from the API call.</returns>
         /// <exception cref="OpenAIClientException" />
-        public async Task<ModerationResponse> ExecuteTextModeration(string text)
+        public virtual async Task<ModerationResponse> ExecuteTextModeration(string text, string? model)
         {
             try
             {
                 using HttpContent content = new StringContent(
                     JsonSerializer.Serialize(new
                     {
-                        model = _options.DefaultModel,
+                        model = model,
                         input = text 
+                    }, new JsonSerializerOptions()
+                    {
+                        DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull,
```

Comment:
Because I'm not a C# developer, could you explain this change? (for my understanding)

Reply:
Will this option, `null` value will be ignored, i.e. the serialization result will be
```json
{
    "input": "text"
}
```
rather than
```json
{
    "model": null,
    "input": "text"
}
```